### PR TITLE
ticdc: Add a description of the compatibility between BR and Lightning

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-faq.md [hunk 1]
```diff
@@ -259,6 +259,14 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
 
 目前 TiCDC 还没完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。
 
+如果有某些表需要使用 TiDB Lightning 或者 BR 恢复数据，可以在这么做：
```
Comment (sdojjy) [inline ticdc/ticdc-faq.md:262]:
```suggestion
如果有某些表需要使用 TiDB Lightning 或者 BR 恢复数据，可以这么做：
```

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -259,6 +259,14 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
...
+
+1. 停止包括这些表的 TiCDC 同步任务。
```
Comment (sdojjy) [inline ticdc/ticdc-faq.md:264]: 应该是删除？与下面的生建对应

Comment (asddongmen) [reply ticdc/ticdc-faq.md:264]: 停止然后 resume 指定 startts 应该也可以？

File: ticdc/ticdc-faq.md [hunk 3]
```diff
@@ -259,6 +259,14 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
 
 目前 TiCDC 还没完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。
 
+如果有某些表需要使用 TiDB Lightning 或者 BR 恢复数据，可以这么做：
+
+1. 停止包括这些表的 TiCDC 同步任务。
+
+2. 使用 TiDB Lightning 或 BR 恢复数据。
```
Comment (flowbehappy) [inline ticdc/ticdc-faq.md:266]: 可以写的更清楚一点，分别恢复上下游的数据

File: ticdc/ticdc-faq.md [hunk 4]
```diff
@@ -259,6 +259,14 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
...
+
+3. 恢复完成后，重新创建 TiCDC 同步任务，从恢复完成的时间点(TSO)开始同步。
```
Comment (flowbehappy) [inline ticdc/ticdc-faq.md:268]: “恢复完成的时间点” 是上游的还是下游的？

File: ticdc/ticdc-faq.md (hunk 2 above)
Comment (flowbehappy) [reply ticdc/ticdc-faq.md:264]: 咱们就写的明确一点吧。如果 pause and resume 可以的话，就写清楚步骤。最好是有命令示例

**Review by flowbehappy — APPROVED** [review]
LGTM

File: ticdc/ticdc-faq.md [hunk 5]
```diff
@@ -258,6 +258,19 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
 ## 为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？
 
 目前 TiCDC 还没完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。
+否则，可能会出现未知的错误，例如 TiCDC 同步卡住、同步延迟大幅增加，或者丢失同步数据等现象。
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:261]:
```suggestion
目前 TiCDC 尚没完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。否则，可能会出现未知的错误，例如 TiCDC 同步卡住、同步延迟大幅增加、或者同步数据丢失。
```

File: ticdc/ticdc-faq.md [hunk 6]
```diff
@@ -258,6 +258,19 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
...
+
+如果有某些表需要使用 TiDB Lightning 或者 BR 恢复数据，可以这么做：
+
+1. 删除包括这些表的 TiCDC 同步任务。
+
+2. 使用 TiDB Lightning 或 BR 在 TiCDC 的上游集群和下游集群分别恢复数据。
+
+3. 恢复完成，并检查了上下游集群对应表的数据一致性之后。使用上游备份的时间点(TSO)作为 TiCDC 同步任务的 `start-ts`，创建新的 TiCDC 同步任务，进行增量同步。例如，假设上游集群的 BR 备份的 snapshot 时间点为 431434047157698561，那么可以使用以下命令创建新的 TiCDC 同步任务：
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:269]:
```suggestion
3. 恢复完成并检查了上下游集群对应表的数据一致性之后。使用上游备份的时间点 (TSO) 作为 TiCDC 同步任务的 `start-ts`，创建新的 TiCDC 同步任务，进行增量同步。例如，假设上游集群的 BR 备份的 snapshot 时间点为 `431434047157698561`，那么可以使用以下命令创建新的 TiCDC 同步任务：
```

File: ticdc/ticdc-faq.md [hunk 7]
```diff
@@ -258,6 +258,19 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
 ## 为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？
 
 目前 TiCDC 还没完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。
+否则，可能会出现未知的错误，例如 TiCDC 同步卡住、同步延迟大幅增加，或者丢失同步数据等现象。
+
+如果有某些表需要使用 TiDB Lightning 或者 BR 恢复数据，可以这么做：
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:263]:
```suggestion
如果有某些使用 TiCDC 同步的表需要使用 TiDB Lightning 或者 BR 恢复数据，可以这么做：
```

File: ticdc/ticdc-faq.md [hunk 8]
```diff
@@ -258,6 +258,19 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
...
+
+1. 删除包括这些表的 TiCDC 同步任务。
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:265]:
```suggestion
1. 删除涉及这些表的 TiCDC 同步任务。
```

File: ticdc/ticdc-overview.md [hunk 9]
```diff
@@ -83,3 +83,4 @@ TiCDC 作为 TiDB 的增量数据同步工具，通过 PD 内部的 etcd 实现
 
 - 暂不支持单独使用 RawKV 的 TiKV 集群。
 - 暂不支持在 TiDB 中[创建 SEQUENCE 的 DDL 操作](/sql-statements/sql-statement-create-sequence.md)和 [SEQUENCE 函数](/sql-statements/sql-statement-create-sequence.md#sequence-函数)。在上游 TiDB 使用 SEQUENCE 时，TiCDC 将会忽略掉上游执行的 SEQUENCE DDL 操作/函数，但是使用 SEQUENCE 函数的 DML 操作可以正确地同步。
+- 暂不支持在同步的过程中对 TiCDC 正在同步的表和库进行 [BR 数据恢复](/br/backup-and-restore-overview.md) 和 [Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入。详情请参考[为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？](/ticdc/ticdc-faq.md#为什么在上游使用了-TiDB-Lightning-和-BR-恢复了数据之后，TiCDC-同步会出现卡顿甚至卡住)。
```
Comment (qiancai) [inline ticdc/ticdc-overview.md:86]:
```suggestion
- 暂不支持在同步的过程中对 TiCDC 正在同步的表和库进行 [BR 数据恢复](/br/backup-and-restore-overview.md) 和 [ TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入。详情请参考[为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？](/ticdc/ticdc-faq.md#为什么在上游使用了-TiDB-Lightning-和-BR-恢复了数据之后，TiCDC-同步会出现卡顿甚至卡住)。
```

File: ticdc/ticdc-faq.md [hunk 10]
```diff
@@ -257,7 +257,19 @@ TiCDC 需要磁盘是为了缓冲上游写入高峰时下游消费不及时堆
 
 ## 为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？
 
-目前 TiCDC 还没完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。
+目前 TiCDC 尚没完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。否则，可能会出现未知的错误，例如 TiCDC 同步卡住、同步延迟大幅增加、或者同步数据丢失。
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:260]:
```suggestion
目前 TiCDC 尚未完全适配 TiDB Lightning 和 BR，请避免在使用 TiCDC 同步的表上使用 TiDB Lightning 和 BR。否则，可能会出现未知的错误，例如 TiCDC 同步卡住、同步延迟大幅增加、或者同步数据丢失。
```

File: ticdc/ticdc-overview.md [hunk 11]
```diff
@@ -83,3 +83,4 @@ TiCDC 作为 TiDB 的增量数据同步工具，通过 PD 内部的 etcd 实现
 
 - 暂不支持单独使用 RawKV 的 TiKV 集群。
 - 暂不支持在 TiDB 中[创建 SEQUENCE 的 DDL 操作](/sql-statements/sql-statement-create-sequence.md)和 [SEQUENCE 函数](/sql-statements/sql-statement-create-sequence.md#sequence-函数)。在上游 TiDB 使用 SEQUENCE 时，TiCDC 将会忽略掉上游执行的 SEQUENCE DDL 操作/函数，但是使用 SEQUENCE 函数的 DML 操作可以正确地同步。
+- 暂不支持在同步的过程中对 TiCDC 正在同步的表和库进行 [BR 数据恢复](/br/backup-and-restore-overview.md) 和 [ TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入。详情请参考[为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？](/ticdc/ticdc-faq.md#为什么在上游使用了-TiDB-Lightning-和-BR-恢复了数据之后，TiCDC-同步会出现卡顿甚至卡住)。
```
Comment (qiancai) [inline ticdc/ticdc-overview.md:86]:
```suggestion
- 暂不支持在同步的过程中对 TiCDC 正在同步的表和库进行 [BR 数据恢复](/br/backup-and-restore-overview.md) 和 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入。详情请参考[为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？](/ticdc/ticdc-faq.md#为什么在上游使用了-TiDB-Lightning-和-BR-恢复了数据之后，TiCDC-同步会出现卡顿甚至卡住)。
```

**Review by qiancai — APPROVED** [review]

File: ticdc/ticdc-overview.md [hunk 12]
```diff
@@ -83,3 +83,4 @@ TiCDC 作为 TiDB 的增量数据同步工具，通过 PD 内部的 etcd 实现
 
 - 暂不支持单独使用 RawKV 的 TiKV 集群。
 - 暂不支持在 TiDB 中[创建 SEQUENCE 的 DDL 操作](/sql-statements/sql-statement-create-sequence.md)和 [SEQUENCE 函数](/sql-statements/sql-statement-create-sequence.md#sequence-函数)。在上游 TiDB 使用 SEQUENCE 时，TiCDC 将会忽略掉上游执行的 SEQUENCE DDL 操作/函数，但是使用 SEQUENCE 函数的 DML 操作可以正确地同步。
+- 暂不支持在同步的过程中对 TiCDC 正在同步的表和库进行 [BR 数据恢复](/br/backup-and-restore-overview.md) 和 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入。详情请参考[为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？](/ticdc/ticdc-faq.md##为什么在上游使用了-tidb-lightning-和-br-恢复了数据之后ticdc-同步会出现卡顿甚至卡住)。
```
Comment (qiancai) [inline ticdc/ticdc-overview.md:86]:
```suggestion
- 暂不支持在同步的过程中对 TiCDC 正在同步的表和库进行 [BR 数据恢复](/br/backup-and-restore-overview.md) 和 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 导入。详情请参考[为什么在上游使用了 TiDB Lightning 和 BR 恢复了数据之后，TiCDC 同步会出现卡顿甚至卡住？](/ticdc/ticdc-faq.md#为什么在上游使用了-tidb-lightning-和-br-恢复了数据之后ticdc-同步会出现卡顿甚至卡住)。
```

**Pull request merged**